src/devhub/components/molecule/NavbarDropdown.jsx

```diff
@@ -0,0 +1,78 @@
+const title = props.title;
```

Comment:
You could remove this from molecule, we have it in organism

Reply:
resolved